cardinal/util.go

```diff
@@ -48,3 +52,34 @@ func isFatalError(err error) bool {
 	}
 	return true
 }
+
+func GetMessage[In any, Out any](wCtx Context) (*MessageType[In, Out], error) {
```

Comment:
This seems to only be used in test and is used to access the `message.In` iterator method which is already exposed via `cardinal.EachMessage`. I don't think we need this

Reply:
Same as above. I'll switch these tomorrow.